controlpanel/api/cluster.py

```diff
@@ -63,6 +73,12 @@ def create(self):
             f"--set=Username={self.user.slug}",
         )
 
+    def reset(self):
```

Comment:
I'll probably leave this comment multiple times but bear with me ☺️  (I'll try not tho 😬 )

Call these this operation just "reset" is a bit ambiguous, I would be more explicit and call it `reset_home`/`ResetHome`/`Reset home` everywhere to avoid any possible confusion.

(For example "old" desktop computers used to have a "reset" button which was restarting the machine so it may lead to someone thinking this has something to do with restarting something)

I know the context in this method is a "User" but I would err on the side of consistency and being explicit everywhere ☺️

Reply:
Aldo, it literally says, "Resets the user's home directory" in the comment directly below this line. :rofl: I'm trying to work out how more explicit I can be. For the record, I really dislike `def long_java_like_methods_that_really_should_be_comments(self):`. I guess this is a question of taste. :wink:

Reply:
Sure if you read the documentation but the thing is in the client code you'll see `ntoll.reset()` but what does that mean? To reset your MFA? To reset your user and its resources to the very initial state where you didn't even have a namespace?

From the Zen of Python:
> Explicit is better than implicit.

So I'd go with `reset_home()` 🙏 . It's not quite `long_java_like_methods_that_really_should_be_comments` ;)